js/main.js

```diff
@@ -0,0 +1,79 @@
+var button = document.getElementById('button');
+
+button.onclick = function displayAndHide() {
+    var x = document.getElementById('displayInfo');
+    if (x.style.display === "") {
+        x.style.display = 'none';
+    } else {
+        x.style.display = '';
+    }
+}
+
+var submitBtn = document.querySelector('#submitBtn'); // Element that is going to call the action
+console.log(submitBtn);
+submitBtn.addEventListener("click", checkName);
+
+function checkName(evt) {
+    evt.preventDefault();
+    var nameArea = document.querySelector("#nameArea");
+    if (nameArea.value == "") {
+        alert("Error: Name is empty!");
+        nameArea.focus();
+        return false;
+    }
+}
+
+var submitBtn = document.querySelector('#submitBtn'); // Element that is going to call the action
```

Comment:
You already have a reference to your 'Submit Button' on line 12 as the 'submitBtn' variable.

You can just reuse this variable :)